src/main/java/org/fcrepo/auth/webac/WebACAuthorizationDelegate.java

```diff
@@ -41,6 +56,24 @@
      */
     private static final Logger LOGGER = LoggerFactory.getLogger(WebACAuthorizationDelegate.class);
 
+    private static final Map<String, String> actionMap;
+
+    static {
+        final Map<String, String> map = new HashMap<>();
+        // WEBAC_MODE_READ Permissions
+        map.put(READ, WEBAC_MODE_READ_VALUE);
+        // WEBAC_MODE_WRITE Permissions
```

Comment:
We probably should include "DELETE" and "UPDATE" as subclasses of "WRITE", as discussed in meetings.